integration-cli/docker_cli_by_digest_test.go

```diff
@@ -403,7 +403,7 @@ func (s *DockerRegistrySuite) TestInspectImageWithDigests(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(imageJSON, checker.HasLen, 1)
 	c.Assert(imageJSON[0].RepoDigests, checker.HasLen, 1)
-	c.Assert(stringutils.InSlice(imageJSON[0].RepoDigests, imageReference), checker.Equals, true)
+	c.Assert(testutil.InSlice(imageJSON[0].RepoDigests, imageReference), checker.Equals, true)
```

Comment:
same for this one (no need to be case-insensitive); I suggest changing the `internal/testutil/stringutils.go` to be case-sensitive, as it looks like we only use it to compare exact values.

Reply:
Modified. except `inslice()`  in `seccomp.go`, I'm not sure about that. Waiting for the feedback.